src/redux/actions/concepts/dictionaryConcepts.js

```diff
@@ -142,10 +142,11 @@ export const fetchDictionaryConcepts = (
       dispatch(populateSidenav());
     }
   } catch (error) {
-    if (error.response) {
+    if (error.response !== undefined) {
       dispatch(isErrored(error.response.data, FETCH_DICTIONARY_CONCEPT));
       dispatch(isFetching(false));
     }
+    notify.show('Network Error. Please try again later!', 'error', 3000);
```

Comment:
I think you can use the function `showNetworkError();` here.

Reply:
working on it

Reply:
@ozezzy , this file doesn't contain the `showNetworkError` function used anywhere.